agent/structs/connect_proxy_config.go

```diff
@@ -174,8 +174,16 @@ type AccessLogsConfig struct {
 	TextFormat string `json:",omitempty" alias:"text_format"`
 }
 
-func (c AccessLogsConfig) ToAPI() api.AccessLogsConfig {
-	return api.AccessLogsConfig{
+func (c *AccessLogsConfig) IsZero() bool {
+	zeroVal := AccessLogsConfig{}
```

Comment:
Should we worry about `c` being nil here? Maybe a non-pointer receiver or return true on nil?

Reply:
Good call. I'll return `true` on `nil`.